src/use-swr-pages.tsx

```diff
@@ -98,14 +98,27 @@ export function useSWRPages<OffsetType = any, Data = any, Error = any>(
 ): pagesResponseInterface<Data, Error> {
   const pageCountKey = `_swr_page_count_` + pageKey
   const pageOffsetKey = `_swr_page_offset_` + pageKey
+  const pageSWRsKey = `_swr_page_swrs_` + pageKey
 
   const [pageCount, setPageCount] = useState<number>(
     cache.get(pageCountKey) || 1
   )
   const [pageOffsets, setPageOffsets] = useState<OffsetType[]>(
     cache.get(pageOffsetKey) || [null]
   )
-  const [pageSWRs, setPageSWRs] = useState<responseInterface<Data, Error>[]>([])
+  const [pageSWRs, setPageSWRs] = useState<responseInterface<Data, Error>[]>(
+    cache.get(pageSWRsKey) || []
```

Comment:
pageSWR's aren't cached currently. We could either just always use an empty array, or cache pageSWRs on line 176.

I tried caching pageSWRs, but this seems to cause a significant delay when loading a pageKey that is cached. On the other hand, if we don't cache pageSWRs, then we lose the scroll position on a page when going back. I haven't found a good solution for the hanging yet.